js/main.js

```diff
@@ -19,10 +19,82 @@ function onSubmit(event) {
     entryNotes: $entryForm.elements.notes.value,
     entryId: data.nextEntryId
   };
+  $entryList.prepend(renderEntry(newEntry));
+  $formDiv.className = data.view + ' hidden';
+  data.view = 'entries';
+  $entriesDiv.className = data.view;
   data.nextEntryId++;
   data.entries.unshift(newEntry);
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
   $entryForm.reset();
 }
 
 $entryForm.addEventListener('submit', onSubmit);
+
+function renderEntry(entryObj) {
+  var listItem = document.createElement('li');
+  var divRow = document.createElement('div');
+  divRow.className = 'row margin-bot-two-rem';
+  listItem.appendChild(divRow);
+
+  var divImgCol = document.createElement('div');
+  divImgCol.className = 'column-half';
+
+  var imgEl = document.createElement('img');
+  imgEl.className = 'img width-100';
+  imgEl.setAttribute('src', entryObj.entryURL);
+
+  var divTextCol = document.createElement('div');
+  divTextCol.className = 'column-half';
+
+  var title = document.createElement('h2');
+  title.className = 'margin-top-half-rem';
+  title.textContent = entryObj.entryTitle;
+
+  var notes = document.createElement('p');
+  notes.textContent = entryObj.entryNotes;
+
+  divRow.appendChild(divImgCol);
+  divImgCol.appendChild(imgEl);
+  divRow.appendChild(divTextCol);
+  divTextCol.appendChild(title);
+  divTextCol.appendChild(notes);
+
+  return listItem;
+}
+
+var $entryList = document.querySelector('.entryList');
+
+function onDOMLoad(event) {
+  for (var entryNum = 0; entryNum < data.entries.length; entryNum++) {
+    var entryRendered = renderEntry(data.entries[entryNum]);
+    $entryList.appendChild(entryRendered);
+  }
+  if (data.view === 'entries') {
+    $formDiv.className = data.view + ' hidden';
+    data.view = 'entries';
+    $entriesDiv.className = data.view;
+  }
+}
+
+window.addEventListener('DOMContentLoaded', onDOMLoad);
+
+var $entriesTab = document.querySelector('.entriesTab');
+var $entriesDiv = document.querySelector('.entries');
+var $newButton = document.querySelector('.newButton');
+var $formDiv = document.querySelector('.entry-form');
+
+function onNewClick(event) {
+  $entriesDiv.className = data.view + ' hidden';
+  data.view = 'entry-form';
+  $formDiv.className = data.view;
+}
+
+function onEntriesClick(event) {
+  $formDiv.className = data.view + ' hidden';
+  data.view = 'entries';
+  $entriesDiv.className = data.view;
+}
```

Comment:
Instead of making an individual function definition for each view, I would make one function named something like `switchViews` that will receive an argument of a `string` containing the name of the view to show. Within that function, you should loop over all the possible views and check if they are the view name given as an argument. If it is that view, you should show it, and if it isn't you should hide it. You would also want to assign a value to `data.view` for whichever view is the correct view. Once you've done this, you'll have a reusable function you can use any time you need to move between views, no matter the context.